### PR TITLE
limitless (new cask)

### PR DESCRIPTION
**Pull request by unitof**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by unitof** [comment]
From the same team as (and likely eventual replacement to) `rewind.rb`

File: Casks/l/limitless.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 :no_check
+
+  url "https://download.limitless.ai/Limitless.dmg"
+  name "Limitless"
+  desc "Personal AI-powered transcription and notetaking service"
+  homepage "https://www.limitless.ai/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
```
Comment (bevanjkay) [inline Casks/l/limitless.rb:12]:
```suggestion
    url "https://storage.googleapis.com/limitless-data/builds/main/manifest.json"
    strategy :json do |json|
      json["version"]
    end
```

File: Casks/l/limitless.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+cask "limitless" do
+  version "0.4006.1,20240422.004237"
```
Comment (bevanjkay) [inline Casks/l/limitless.rb:2]:
```suggestion
  version "0.4006.1"
```

File: Casks/l/limitless.rb (hunk 1 above)
Comment (unitof) [reply Casks/l/limitless.rb:12]: For my future reference, how did you find this URL? Any standard place to look or inspection tool?

Charles proxy got me nowhere with this app as it doesn't seem to phone home for updates.

Comment (bevanjkay) [reply Casks/l/limitless.rb:12]: I used Proxyman, which is essentially the same as Charles, it just works a bit better for me.

**Comment by unitof** [comment]
Oops, was testing GitHub app and didn't realize it committed each review. Feel free to squash as desired or I will when back at desktop.

**Review by bevanjkay — APPROVED** [review]
Thanks @unitof

**Pull request merged**